### ISSUE TITLE
Keyboard dialog updates for both visual and accessible content

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#214]
@zepumph, just noticed a few updates that will be needed for KB dialog:

**Updates to Visuals:**
- Change order of Move and Jump details (heading and list items)
- Insert "and release ruler" in the [J]+[H] instructions block.
![Screen Shot 2019-11-27 at 1 47 02 PM](https://user-images.githubusercontent.com/1902116/69746556-1445d280-111f-11ea-9a86-7c58cc51ab22.png)

**Updates to A11y Strings for Keyboard Dialog**
- remove "the" in first line under "Move Spheres", the bullet item reads: 
  - "Move sphere left and right with Left and Right Arrow keys."
- Reverse order of "Move" and "Jump" in heading for moving ruler
  - change to: "Jump or Move Grabbed Ruler"
- Insert "and release ruler" to JH instructions
  - change to: "Jump and release ruler to home position with J plus H. "
- Reorder the bullets, the last two are first and the first two are last (see adjusted screenshot)

![Screen Shot 2019-11-27 at 1 46 36 PM](https://user-images.githubusercontent.com/1902116/69747529-1dd03a00-1121-11ea-85c2-24bfd8ab272f.png)

**Comment by zepumph** [comment]
Implemented above, please review.

**Comment by terracoda** [comment]
@zepumph, sorry I missed one detail for the KB Shortcuts dialog.
We now have a radio button group.
We need to use the Basic Actions options that include the words "or group" like in BASE:
- Move to next item or group  [Tab]
- Move to previous item or group with [Shift] + [Tab]

**A11y Content for these same items**
- Move to next item or group with Tab key.
- Move to previous item or group with Shift plus Tab key.
- Move through items in a group with Up and Down arrow keys.

I don't know if there is an option for this, or if you handle it manually. It would be good to have an option as not all sims have groups in them. Also, it seems like we have some inconsistency. At the moment BASE has "or group" in the visual dialog, but not in the A11y content and MAL is missing the "or group" wording in both places.

Note there is no corresponding visual for the "Move through items in a group". I will reach out to Emily and Amy about that. I can't remember if there was a decision not to have that.

I'll update the text in the design doc. Sorry, I missed this.

**Comment by zepumph** [comment]
>I don't know if there is an option for this

We do have an option, see the commit above (very easy thanks to @jessegreenberg).

I also did this for CL. How does that sound? I can make further changes, but they would be in common code. If you think that something should change from how it is in CL and GFL right now, then I would recommend making an issue about it in `scenery-phet` and it can apply to all sims.

**Comment by terracoda** [comment]
@zepumph, nice there was an option, but I don't see the added words "item or group" in the A11y View of the dialog. Should this comment move over to scenery-phet?

- [ ] add "item or group" to a11y content. First 2 bullets in https://github.com/phetsims/gravity-force-lab/issues/214#issuecomment-570013622

AND ONE more visual thing
- [ ] I just noticed an extra "Jump or" string in the visual dialog, please remove.

![Screen Shot 2020-01-08 at 6 56 58 PM](https://user-images.githubusercontent.com/1902116/72021997-fb662c00-3249-11ea-93d2-e18955af84fd.png)

**Comment by zepumph** [comment]
I removed the extra jump, I also fixed a regression where the group content was showing up in BASICS, and I created https://github.com/phetsims/scenery-phet/issues/570 for solving the addition of group related strings generally (which has been implemented and is out for review). 

@terracoda anything else here?

**Comment by terracoda** [comment]
All verified. Closing.
